libs/martech/helpers.js

```diff
@@ -528,7 +567,17 @@ export const createRequestUrl = ({
   hitType,
 }) => {
   const TARGET_API_URL = getUrl(hitType === 'propositionDisplay');
-  dataStreamId = env === 'prod' ? DATA_STREAM_IDS_PROD.default : DATA_STREAM_IDS_STAGE.default;
+  const { hostname } = window.location;
+  if (hostname.includes('business.adobe')) {
+    dataStreamId = DATA_STREAM_IDS_PROD.business;
+  } else if (
+    hostname.includes('business.stage.adobe')
+    || hostname.includes('bacom--adobecom.hlx')
```

Comment:
This should be .aem, right? Hlx is being deprecated. Or to be safe, you could include both.
```suggestion
    || hostname.includes('bacom--adobecom.hlx')
    || hostname.includes('bacom--adobecom.aem')
```

Reply:
Done. Thanks! @meganthecoder